Automattic/sensei
# Use WordPress to register/login instead of built-in functions

Fixes https://github.com/Automattic/sensei/issues/1202. This PR is supposed to remove the registration/login form from `my-courses`. Instead, it should redirect to wp-login. The reasoning for this is given in the issue, i.e. no confirmation email is sent, SSO support, etc.

Test case:
1. Navigate to a page that has either `usercourses` or `sensei_user_courses` as a shortcode, e.g. `http://localhost/wordpress/my-courses`.
Expected: You should be redirected to the WP login screen
2. Login to WP
Expected: You should be redirected back to the original page, e.g. `my-courses`.
3. Reload
Expected: Now that you're logged in, you should see `my-courses` contents
